src/snowflake/cli/_plugins/snowpark/commands.py

```diff
@@ -225,8 +228,8 @@ def build_artifacts_mappings(
             entities_to_imports_map[entity_id].add(artefact_dto.import_path(stage))
         stages_to_artifact_map[stage].update(required_artifacts)
 
-        if project_paths.dependencies.exists():
-            deps_artefact = project_paths.get_dependencies_artefact()
+        deps_artefact = project_paths.get_dependencies_artefact()
```

Comment:
nit: we should pick one of british or american spelling for artifact/artefact and use it consistently

Reply:
I agree, created ticket for it.